lib/infoblox.rb

```diff
@@ -13,7 +13,7 @@
 end
 
 module Infoblox
-  WAPI_VERSION   = '1.0'
+  WAPI_VERSION   = '1.1'
```

Comment:
@epssy This will break the gem for people who are not able to use 1.1, no? We should make this configurable.